src/main/java/dk/aau/cs/model/CPN/ColorType.java

```diff
@@ -40,10 +40,14 @@ public Integer size() {
         return colors.iterator();
     }
 
-    public Vector<Color> getColors(){
+    public Vector<Color> getColors() {
         return colors;
     }
 
+    public String getSemanticValue() {
```

Comment:
Well, this is actually the same as before (still comparing strings) and in fact I would prefer not to have this conversion to string as a new method doing just .toString(); anyway.  What I had in mind was whether one could implement a method returning the range (as a pair of integers) and compare these numbers to check if an identical integer range already exists. But I am not sure if this is easy to implement, if not than the (original) string comparison method could be also acceptable.